sale_invoice_group_method/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Sale Invoice Group Method",
+    "summary": "This module allows you to combine several"
+    "Sales Orders into a single invoice,"
+    "if they meet the group criteria defined by the "
+    "'Invoice Group Method'",
+    "author": "Eficent, Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "category": "Sale Workflow",
+    "website": "https://github.com/OCA/sale-workflow",
+    "license": "LGPL-3",
+    "depends": [
+        "account",
+        "sale_order_action_invoice_create_hook",
```

Comment:
@dantiller2 This module seems to be not yet merged. Could you point to its migration PR in this PR description ? Thanks